cpp/CMakeLists.txt

```diff
@@ -59,14 +59,18 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
-include(PythonUtils) # Must be called before Pybind (third_party) to override its finding mechanism
+if(NOT WIN32)
```

Comment:
Is this still necessary given what we've learned the real cause was?

Reply:
Yes, it had issues discovering the Python static lib with this left in.